package.json

```diff
@@ -11,7 +11,9 @@
     "pretest": "npm run lint",
     "test": "npm run unit-test",
     "type-check": "tsc",
-    "unit-test": "istanbul cover node_modules/mocha/bin/_mocha tests/lib/**/*.js tests/util/**/*.js tests/index.js"
+    "unit-test": "istanbul cover node_modules/mocha/bin/_mocha tests/lib/**/*.js tests/util/**/*.js tests/index.js",
+    "generate-list-of-rules": "./scripts/generate-list-of-rules.js README.md",
```

Comment:
this would be great as a published package, so we can use it in eslint-plugin-react and eslint-plugin-jsx-a11y as well :-D (but we should wait until this PR lands to do that)